### PR TITLE
Ocrvs 2898 Show Radio and Select options content keys

### PR DESCRIPTION
**Pull request by sadmananik**

### CODE REVIEW

**Comment by sadmananik** [comment]
ocrvs-2898

File: packages/client/src/components/formConfig/formTools/DefaultFieldTools.tsx [hunk 1]
```diff
@@ -124,7 +124,11 @@ export function DefaultFieldTools({
           {intl.formatMessage(messages.contentKey)}
           <StyledTooltip />
         </Subtitle>
-        <Body>{contentKey}</Body>
+        <Body>
+          {contentKey.map((content, index) => (
+            <div key={index}>{content}</div>
```
Comment (Zangetsu101) [inline packages/client/src/components/formConfig/formTools/DefaultFieldTools.tsx:129]:
```suggestion
            <span key={index}>{content}</span>
```
Span is probably a better choice here

File: packages/client/src/forms/configuration/configFields/utils.ts [hunk 2]
```diff
@@ -139,7 +140,19 @@ export function getFieldDefinition(
...
 
 export function getContentKey(formField: IFormField) {
-  return formField.label.id
+  if (
+    (formField.type === 'RADIO_GROUP' ||
+      formField.type === 'RADIO_GROUP_WITH_NESTED_FIELDS' ||
+      formField.type === 'SELECT_WITH_OPTIONS' ||
+      formField.type === 'DOCUMENT_UPLOADER_WITH_OPTION') &&
+    !['country', 'countryPermanent', 'nationality'].includes(formField.name)
+  ) {
```
Comment (Zangetsu101) [inline packages/client/src/forms/configuration/configFields/utils.ts:149]: We should use the CONSTANTS defined here instead https://github.com/opencrvs/opencrvs-core/blob/fb0985a913ca955f57f3aac87204f83e178f0cb5/packages/client/src/forms/index.ts#L37-L62

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: packages/client/src/components/formConfig/formTools/DefaultFieldTools.tsx (hunk 1 above)
Comment (sadmananik) [reply packages/client/src/components/formConfig/formTools/DefaultFieldTools.tsx:129]: Agree

**Pull request merged**